app/assets/stylesheets/base/_normalize.scss

```diff
@@ -0,0 +1,26 @@
+.some {
+  .bad {
+    .nesting {
+      .thats {
+        .sure {
+          .to {
+            .cause {
+              .linter {
+                .violations {
+                  .but {
+                    .should {
+                      .be {
+                        .ignored {
```

Comment:
Empty rule<br>Selector should have depth of applicability no greater than 2, but was 13